### PR TITLE
Add nextflow docs (Bessemer)

### PR DESCRIPTION
**Pull request by Carldkennedy**
Related to #1621 #1553 

Draft docs.  With current staging install directories. **Needs updating once Live**

- [x] Install in Live
- [x] Edit docs to reflect Live

### CODE REVIEW

**Comment by Carldkennedy** [comment]
[5_tips_for_hpc_users](https://www.nextflow.io/blog/2021/5_tips_for_hpc_users.html
)
[5-more-tips-for-nextflow-user-on-hpc](https://www.nextflow.io/blog/2021/5-more-tips-for-nextflow-user-on-hpc.html)

**Review by kugzy — APPROVED** [review]
LGTM

**Review by nikishe — COMMENTED** [review]
Sorry , I was logged into other account. Looks good to me

**Review by nikishe — APPROVED** [review]

**Comment by Carldkennedy** [comment]
> Is the idea to pull in some of the content from #1621?

@willfurnass Yes. Currently only in staging. Plan to update once live.

**Review by nikishe — COMMENTED** [review]
LGTM

**Comment by willfurnass** [comment]
@Carldkennedy Still working on this?

**Comment by jkwmoore** [comment]
Mark as ready if/when not a draft and I'll give it a review.

**Comment by Carldkennedy** [comment]
> @Carldkennedy Still working on this?

Was on hold during the Stanage sprint. Finished now

File: bessemer/software/apps/nextflow.rst [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    #SBATCH --mem=1000
+    #SBATCH --job-name=tutorial_smp_2
+    #SBATCH --output=tutorial_smp_2
+    #SBATCH --time=00:01:00
+    #SBATCH --mail-user=a.person@sheffield.ac.uk
+    #SBATCH --mail-type=ALL
+    module load Nextflow/22.04.0
+    nextflow run tutorial.nf
+     
+You can now submit this job to the SLURM scheduler with
```
Comment (nikishe) [inline bessemer/software/apps/nextflow.rst:120]:
```suggestion
You can now submit this job to the SLURM scheduler with:
```

**Pull request merged**